.github/workflows/e2e-tests.yml

```diff
@@ -162,23 +162,23 @@ jobs:
           -v "$(pwd)/e2e_tests":/e2e_tests e2e_tests
 
     - name: Upload test results
-      uses: actions/upload-artifact@v4
+      uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02
```

Comment:
you should be able to comment the version like in https://github.com/dependabot/dependabot-core/issues/4691 which is useful